src/arch/xtensa/Kconfig

```diff
@@ -8,4 +8,26 @@ config SMP
 	help
 	  Build SMP instead of UP
 
+config CORE_COUNT
+	int "Number of cores"
+	range 1 4
+	default 1
+	depends on CAVS
+	help
+	  Number of cores
+
+config HP_MEMORY_BANKS
```

Comment:
While the number of cores is cpu arch related, the L2 memory is platform specific, interfaced with cpu through PIF. So move the memory defined into the CAVS submenu.